### PR TITLE
ARS-358 Components & Happy Path

### PR DESCRIPTION
**Pull request by RawToast**
https://user-images.githubusercontent.com/8013000/209976566-da7e7b5b-ebaa-4788-8462-0a901e1133b4.mov

### CODE REVIEW

File: app/views/ImportGoodsView.scala.html [hunk 1]
```diff
@@ -28,6 +28,9 @@
 
     @formHelper(action = routes.ImportGoodsController.onSubmit(mode), 'autoComplete -> "off") {
 
+         <p class="govuk-caption-xl hmrc-caption-x"><span class="govuk-visually-hidden">
+            This section is </span>@messages("importGoods.caption")</p>
```
Comment (C-Thomp) [inline app/views/ImportGoodsView.scala.html:32]: needs to go to messages file // welsh translation

File: app/views/ImportingGoodsView.scala.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+            id = "help_and_support_link",
+            text = messages("importingGoods.point.5.link"),
+            call = Call("GET", config.helpAndSupportUrl)
+        )
+    }
+
+@layout(pageTitle = titleNoForm(messages("importingGoods.title"))) {
+
+    <p class="govuk-caption-xl hmrc-caption-x"><span class="govuk-visually-hidden">
+        This section is </span>@messages("importingGoods.caption")</p>
```
Comment (C-Thomp) [inline app/views/ImportingGoodsView.scala.html:74]: caption component?

File: app/views/components/Link.scala.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this()
+
+@(id: String, text: String, call: Call, newTab: Boolean = false, classes: String = "govuk-link govuk-link--no-visited-state")
```
Comment (C-Thomp) [inline app/views/components/Link.scala.html:19]: Default true

File: app/views/ImportGoodsView.scala.html (hunk 1 above)
Comment (RawToast) [reply app/views/ImportGoodsView.scala.html]: https://github.com/hmrc/advance-valuation-rulings-frontend/pull/7/commits/580660c4f141b67f2b09e5aa5e8a75198b8d916d

File: app/views/ImportingGoodsView.scala.html (hunk 2 above)
Comment (RawToast) [reply app/views/ImportingGoodsView.scala.html]: https://github.com/hmrc/advance-valuation-rulings-frontend/pull/7/commits/580660c4f141b67f2b09e5aa5e8a75198b8d916d

File: app/views/components/Link.scala.html (hunk 3 above)
Comment (RawToast) [reply app/views/components/Link.scala.html]: https://github.com/hmrc/advance-valuation-rulings-frontend/pull/7/commits/580660c4f141b67f2b09e5aa5e8a75198b8d916d

**Review by C-Thomp — APPROVED** [review]
Wunderbar

**Pull request merged**